tests/appsec/iast/test_iast.py

```diff
@@ -56,3 +67,19 @@ def test_insecure_md5_hash(self):
         r = self.weblog_get("/iast/insecure_hashing/test_md5_algorithm")
 
         interfaces.library.expect_iast_vulnerabilities(r, vulnerability_type="WEAK_HASH", evidence="md5")
+
+    def test_secure_sql(self):
```

Comment:
@missing_feature(library="nodejs", reason="Need to implement nodejs endpoint")

Reply:
Changed, thanks!